ts/src/base/functions/customThrottle.ts

```diff
@@ -86,35 +85,44 @@ class Throttler {
             rejecter = reject;
         });
 
+        const capacity = Object.keys (this.priorityQueue).reduce (
+            (acc, cur) => acc + this.priorityQueue[cur].length,
+            0
+        );
         // TODO manage capacity for both queues
-        if (this.queue.length > this.config['maxCapacity']) {
+        if (capacity > this.config['maxCapacity']) {
             throw new Error ('throttle queue is over maxCapacity (' + this.config['maxCapacity'].toString () + '), see https://github.com/ccxt/ccxt/issues/11645#issuecomment-1195695526');
         }
         cost = (cost === undefined) ? this.config['cost'] : cost;
 
         let priority;
-        if (customPriority !== undefined) {
+        if (customPriority !== undefined && customPriority in CustomThrottlePriority) {
             priority = customPriority;
-        } else if (path !== undefined) {
+        } else if (path !== undefined && this.config['priorities'][path] !== undefined
+        ) {
             priority = this.config['priorities'][path];
+        } else {
+            priority = CustomThrottlePriority.MEDIUM;
         }
```

Comment:
If I understand correctly :

1st case: when we call the function with the priority index in params
2nd case: we didn't put the priority inde in the call, but it's the config of ccxt (added when instatiating the exchange like `new ccxt.binance...`
3rd case: default priority is Medium

Is that right? Maybe would be interesting to add few comments